db/functions/js_callbacks.go

```diff
@@ -91,15 +92,16 @@ func (runner *jsRunner) do_func(funcName string, params map[string]any, sudo boo
 
 // Implementation of JS `user.get(docID, docType)` function
 func (runner *jsRunner) do_get(docID string, docType *string, sudo bool) (any, error) {
-	if err := runner.currentDB.CheckTimeout(runner.ctx); err != nil {
+	if err := db.CheckTimeout(runner.ctx); err != nil {
 		return nil, err
 	}
 	if sudo {
 		user := runner.currentDB.User()
 		runner.currentDB.SetUser(nil)
 		defer func() { runner.currentDB.SetUser(user) }()
 	}
-	rev, err := runner.currentDB.GetRev(runner.ctx, docID, "", false, nil)
+	collection := runner.currentDB.GetSingleDatabaseCollectionWithUser()
```

Comment:
It would be better to define currentCollection on jsRunner, and initialize in jsRunner.CallWithDB, as opposed to doing it every callback.

Reply:
Agree to leave this for now, needs review of user mutability in js_callbacks.

Reply:
I think this should be default collection for the time being, until collection support gets added to the functions/graphQL API.